.github/workflows/dev.yml

```diff
@@ -103,6 +103,28 @@ jobs:
       - name: Test
         run: cd build && ctest -j3 --output-on-failure
 
+  dodo:
+    # Tests with: Autconf on oldest supported Ubuntu (in non-extended support)
+    name: GCC -Os, old Autotools
+    runs-on: ubuntu-20.04
```

Comment:
Annoyingly... the changes between Automake 1.15 and 1.16 turn out to be somewhat annoying, and it took several attempts to make my changes to Makefile.am. We didn't have an autoconf build on old Ubuntu, which would have helped me to catch the problems.